lib/strings.nix

```diff
@@ -472,18 +471,7 @@ rec {
     ["&quot;" "&apos;" "&lt;" "&gt;" "&amp;"];
 
   # Obsolete - use replaceStrings instead.
```

Comment:
Might as well convert it into a throw if the fallback is removed.

Reply:
replaceChars warn moved to https://github.com/NixOS/nixpkgs/pull/205701